pokemongo_bot/utils.py

```diff
@@ -145,3 +145,5 @@ def convert_to_utf8(data):
         return list(map(convert_to_utf8, data))
     elif isinstance(data, set):
         return set(map(convert_to_utf8, data))
+    else:
+        return str(data)
```

Comment:
There was an issue when the utf8 encoding the json where the pokemon unique ID was being stripped out as it didn't match in this method.